vision/snippets/quickstart/quickstart_test.py

```diff
@@ -15,7 +15,7 @@
 import quickstart
 
 
-def test_quickstart() -> None:
-    labels = quickstart.run_quickstart()
-    for label in labels:
-        assert len(label.description) > 0
+def test_quickstart(capsys) -> None:
```

Comment:
Recently we've been updating multiple samples to stop using capsys and add return values to individual functions. It seems that you're doing the opposite here. I suggest to keep returning the labels and use them in tests.

Reply:
Hmm, that practice doesn't align with the Samples Style guide:
https://googlecloudplatform.github.io/samples-style-guide/#result

Reply:
I have reached out via chat to discuss this in context of [another PR](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/9323#discussion_r1260892266) with similar request.

Reply:
Since a related docs change has already been submitted, I'll accept the requested changes.